### PR TITLE
Add asg-scaling-activities()

### PR DESCRIPTION
**Pull request by jonhiggs**

### CODE REVIEW

File: lib/asg-functions [hunk 1]
```diff
@@ -169,4 +169,17 @@ asg-suspend() {
   done
 }
 
+asg-scaling-activities() {
+  # type: query
+  # show all scaling activities
+  local inputs=$(__bma_read_inputs $@)
+  local output=$(__bma_read_switches $inputs | grep ^--output | cut -d\  -f2-)
+  [[ -z "$inputs" ]] && __bma_usage "auto-scaling-group" && return 1
+  for asg in $(__bma_read_resources ${inputs}); do
```
Comment (galaxy4public) [inline lib/asg-functions]: I'd suggest to follow some kind of a convention when we are working with variable names.  For example on line 176 there is "$inputs", but on line 178 there is "${inputs}".  I propose not ti use curly braces unless it is required (e.g. for expansions or when the variable is in the middle of a word).  Also, the code is very inconsistent with the quotation of variables, e.g. on line 177 "$inputs" are properly double quoted, but on other lines variables are passed unquoted.  It is quite important to follow strict quotation rules or we may get some unexpected surprises down the line. WDYT?

**Comment by jonhiggs** [comment]
I totally agree about the inconsistency in style and would like to see us converge upon a standard. I totally agree with always quoting unless you explicitly want the behavior of not quoting. I prefer to alway use the curly braces unless its a single letter var like $1 or $i. I'll happily be convinced otherwise though.

File: lib/asg-functions (hunk 1 above)
Comment (mbailey) [inline lib/asg-functions]: Thanks for raising this. I'll create a pull request for doc/style.md

**Comment by mbailey** [comment]
@galaxy4public How would you suggest this line should read:
```
local inputs=$(__bma_read_inputs $@)
```

**Comment by mbailey** [comment]
We're deferring decision on styleguide for now. Agreed that we should have one.

Approved subject to Mike testing it.

**Pull request closed without merging**